frontend/src/components/ui/AppFacilityFilter.vue

```diff
@@ -0,0 +1,54 @@
+<template>
```

Comment:
This is a pretty specific component and goes beyond a ui wdiget. I'd just put in under facilities/FacilityFilter.vue.

Reply:
Fixed.